### ISSUE TITLE
  [Feature Request] A Mini (maps) endpoint of pokemon world locations

### ISSUE THREAD

**Issue by The-Programador** [PokeAPI/pokeapi#726]

**Comment by The-Programador** [comment]
Um endpoint de Mini Mapa para exibir a região de um determinado pokémon

**Comment by The-Programador** [comment]
YES @Naramsim

**Comment by The-Programador** [comment]
@Naramsim 
A Mini Map endpoint to display the region of a given pokémon, I believe it's pretty cool, the request occurs through the pokémon's ID or name and brings in the request a minimap of the habitat or region of the pokémon in question

**Comment by merfed** [comment]
Might be near impossible to include anything for gen-8.

**Comment by The-Programador** [comment]
> We already have locations for pokemons.
> 
> So we could link these locations to some images. That would work fine. But someone needs to scrape the images and link them

@Naramsim 

would be perfect, is it feasible, hey man, will you include this functionality?

**Comment by SimplyBLGDev** [comment]
This is hardly feasible and *should* fall outside the responsibilities of the API itself, you'll need a client that processes the data to generate the image, these images **should not** be part of the API as they would have little connection to the rest of the data *and* is able to be calculated with already available data.
something _similar_ to this functionality exists already for most generations at https://simplyblgdev.github.io/Pokemon/ which uses PokeAPI data to generate that info.
@The-Programador @Naramsim
I suggest we close this issue.

**Comment by phalt** [comment]
I tend to agree, it feels like something you'd build on top of data we supply.
